src/utils.js

```diff
@@ -312,7 +314,7 @@ export function toIdSchema(schema, id, definitions) {
   return idSchema;
 }
 
-export function parseDateString(dateString) {
+export function parseDateString(dateString, includeTime = true) {
```

Comment:
Nit: wouldn't it better to have `parseDateString()` and `parseDatetimeString()` instead ?